frontend-web/webclient/app/UCloud/ProvidersApi.tsx

```diff
@@ -92,13 +108,67 @@ class ProviderApi extends ResourceApi<Provider, Product, ProviderSpecification,
         }
     };
 
+    public retrieveOperations(): Operation<Provider, ResourceBrowseCallbacks<Provider> & ProviderCallbacks>[] {
+        return [
+            {
+                text: "Create " + this.title.toLowerCase(),
+                icon: "upload",
+                color: "blue",
+                primary: true,
+                canAppearInLocation: loc => loc !== "IN_ROW",
+                enabled: (selected, cb) => {
```

Comment:
Should only be enabled if you are a UCloud admin (you can use this UI if you have edit rights for the provider)